README.md

```diff
@@ -277,18 +275,17 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   ternary operators must not be nested. Prefer if/else constructs in these
   cases.
 
+* Avoid multiline `?:` (the ternary operator); use `if/unless` instead.
+
 * Use `when x then ...` for one-line cases.
 
 * Use `!` instead of `not`.
 
-* Prefer `&&`/`||` over `and`/`or`. [More info on `and/or` for control
-  flow](http://devblog.avdi.org/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/).
```

Comment:
Removed the link as I find it unnecessary.

Reply:
Not sure, I think it's interesting. Like I vaguely remember that there was some weird difference esp. folks new to Ruby might be like "but why?"

Reply:
The [difference involves precedence](https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby), but it's not too hard to search for. Still, it might make sense to still have a link.